scss/sections/_home.scss

```diff
@@ -11,30 +10,41 @@
   background-image: url("../images/profilepic/background.jpg");
```

Comment:
- [ ] The pages should use the exact same colors, font, backgrounds, borders, etc, from the original design. You don't have to use exact same background but it will be better if you use a background with the similar color as a given design template.

Reply:
@sinansevgi Hello, I agree with everything that you suggested and I will work to fix them. In regards to making the pictures, a compulsory requirement is not something I see in the requirements. We are not given any photos to work with and I was not able to find any photos that match. Making this optional would be nice, thank you.

Reply:
Sorry if i am not clear about my comment. You don't have to use same images. You can use your image too. But the background color (overall color of the image) should be similar.